src/QirRuntime/test/QIR-static/qir-test-output.qs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR.Out {
+
+    open Microsoft.Quantum.Intrinsic;
+
+    function MessageTest(msg: String) : Unit {
+        Message( msg );
```

Comment:
Nit: should be no spaces around params in a call

Reply:
BTW, if you run clang formatter fixes this?

Reply:
Copy-paste cost... ;-) fixed.
I'll try clang-format and see how it works.

Reply:
Here is the output of 
> qsharp-runtime\src\QirRuntime\test\QIR-static>**clang-format --style=file qir-test-output.qs** 

:-)
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

namespace Microsoft.Quantum.Testing.QIR.Out
{

    open Microsoft.Quantum.Intrinsic;

    function MessageTest(msg
                         : String)
        : Unit
    {
        Message(msg);
    }

} // namespace Out
```
So yes, _it does remove the spaces inside of parentheses_, but based on our "qsharp-runtime/src/QirRuntime/**.clang-format** ", which is C++-oriented I believe, it _does a number of undesirable changes_ to the .qs file, including the change of a comment in the last line. ;-)

Reply:
@samarsha has been working on developing a Q# formatter, we should consider using that has been packaged as a redeployable tool (https://github.com/microsoft/qsharp-compiler/issues/864).